frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -158,13 +159,17 @@ class Sidebar extends Component<Props> {
       <div className={['sidebarContainer', hidden ? 'forceHide' : ''].join(' ')}>
         <div className={['sidebarOverlay', activeSidebar ? 'active' : ''].join(' ')} onClick={toggleSidebar}></div>
         <nav className={['sidebar', activeSidebar ? 'forceShow' : '', shown ? 'withGuide' : ''].join(' ')}>
-          <Link
-            to={accounts.length ? '/account-summary' : '/'}
-            onClick={this.handleSidebarItemClick}>
-            <div className="sidebarLogoContainer">
+          <div className="flex flex-between sidebarLogoContainer">
+            <Link
+              to={accounts.length ? '/account-summary' : '/'}
+              onClick={this.handleSidebarItemClick}>
               <AppLogoInverted className="sidebarLogo" />
-            </div>
-          </Link>
+            </Link>
+            <button className="closeButton" onClick={toggleSidebar}>
```

Comment:
I just realized this component doesn't use CSS module and the CSS is all global. This component should probably use CSS module and use `className={style.closeButton}`. It would take renaming `sidebar.css` to `sidebar.module.css` and then import style from sidebar.module.css. But can be in a different PR or 2nd commit, up to you.

Reply:
Haha, it seems like only this component isn't using css module 😄 https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/web/src/style/index.css

Will migrate it to be using one here.

Reply:
wait the styles in `src/style/..` are supposed to be global.
but all css in `src/components/*` are supposed to be CSS modules